servers/shared-snowplow-consumer/src/snowplow/EventHandler.ts

```diff
@@ -13,8 +9,7 @@ export class EventHandler {
   }
 
   public trackObjectUpdate<T = any>(
-    tracker: Tracker,
-    objectUpdate: ObjectUpdate & CommonEventProperties<T>,
+    ...[tracker, objectUpdate]: Parameters<typeof baseTrackObjectUpdate<T>>
```

Comment:
use conditional parameters and propagate the generics to get proper argument types without editing the generated file (CommonEventProperties was not exported)

Reply:
smart. this is much better then what i did before